### PR TITLE
fix: blocks annoyance popup only

### PR DESCRIPTION
**Pull request by HenryQW**
***Description***:
* **Current behaviour**: 

All popups are blocked, including news.

* **Expected behaviour**: 

Only blocks annoyance popup, but allows other normal content popups.

<details><summary>Screenshot:</summary>

image

</details><br/>

### CODE REVIEW

File: SocialFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -53,7 +53,7 @@ freundin.de#$#.item-content__content { padding-top: 0px !important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/47027
 dailywire.com#$#.SocialMediaShareButton { visibility: hidden!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/44288
-simplywall.st#$#div[data-cy-id^="modal-"] { display: none!important; }
+simplywall.st#$#div[data-cy-id="modal-ModalPortal-0-"] { display: none!important; }
```
Comment (Alex-302) [inline SocialFilter/sections/general_extensions.txt:56]: please remove space at the end of the line

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Comment by Alex-302** [comment]
Hi. Which popups are useful? I see only this.

**Comment by HenryQW** [comment]
> Hi. Which popups are useful? I see only this.

Pretty much every "View Data" button on the page will trigger a modal popup, which will be blocked by the current rule. These buttons are located in Section 3, 5, 7 etc.

image

image

**Comment by Alex-302** [comment]
Got it.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**